### ISSUE TITLE
Should faq.html updated to mentions the creation of a `.openjfx` directory under the user directory?

### ISSUE THREAD

**Issue by e-Gyi-qO** [sparrowwallet/sparrow#1539]
Can it be that a `.openjfx` directory was recently created at my user directory because of sparrowwallet?
I used the suggestions at [faq.html#im-running-on-a-high-resolution-screen-and-sparrow-icons-and-text-are-tiny](https://sparrowwallet.com/docs/faq.html#im-running-on-a-high-resolution-screen-and-sparrow-icons-and-text-are-tiny), and [faq.html#sparrow-is-too-big-for-my-screen](https://sparrowwallet.com/docs/faq.html#sparrow-is-too-big-for-my-screen). Which boils to suggesting `_JAVA_OPTIONS=-Dglass.gtk.uiScale=...`. Does such usage, or maybe any usage of sparrowwallet, would lead to the creation of a `.openjfx` directory? If the answer is yes, can the [faq.html](https://sparrowwallet.com/docs/faq.html) point that out?
Reference: [/JabRef/jabref/issues/5817](/JabRef/jabref/issues/5817).

**Comment by craigraw** [comment]
Yes, this is standard behaviour in JavaFX. The location of the `.openjfx` folder (into which is extracted the native libraries JavaFX requires) can be changed, for example
```
JAVA_OPTIONS=-Djavafx.cachedir=/another/folder /opt/sparrow/bin/Sparrow
```

It might be possible to include these libraries in the application installation instead, avoiding this. I'll rename and keep this issue open to investigate.

**Comment by craigraw** [comment]
Also see https://bugs.openjdk.org/browse/JDK-8316276
